includes/admin/class-amp-cache-utilities.php

```diff
@@ -0,0 +1,251 @@
+<?php
+abstract class AMP_Cache_Utilities {
+
+	static $amp_valid_content_types = array( 'c', 'i', 'r' );
```

Comment:
I'm surprised this passed tests under PHP 5.2. I was under the impression that properties couldn't be defined as arrays in the class definition. Maybe it's different for static properties?